build/Targets/Settings.props

```diff
@@ -74,11 +73,18 @@
          https://github.com/dotnet/sdk/issues/1083#issuecomment-292013375 
 
          This needs to be conditional to support a fresh restore -->
-    <!-- <RoslynNetSdkRootPath>$(NuGetPackageRoot)\Microsoft.Net.Sdk\$(MicrosoftNetSdkVersion)\Sdk\</RoslynNetSdkRootPath>  -->
-    <RoslynNetSdkRootPath>$(MSBuildSDKsPath)\Microsoft.Net.Sdk\Sdk\</RoslynNetSdkRootPath> 
+    <!-- <RoslynNetSdkRootPath>$(NuGetPackageRoot)\Microsoft.NET.Sdk\$(MicrosoftNetSdkVersion)\Sdk\</RoslynNetSdkRootPath>  -->
+    <RoslynNetSdkRootPath>$(MSBuildSDKsPath)\Microsoft.NET.Sdk\Sdk\</RoslynNetSdkRootPath> 
     <RestoreProjectStyle>PackageReference</RestoreProjectStyle>
   </PropertyGroup>
 
+  <ItemGroup Label="ToolsetPackages">
+    <PackageReference Include="$(ToolsetCompilerPackageName)">
+      <Version>$(ToolsetCompilerPackageVersion)</Version>
+      <ExcludeAssets>all</ExcludeAssets>
+    </PackageReference>
+  </ItemGroup>
```

Comment:
Why do we need this?  It's adding an implicit PackageReference to every item that build in our repo.  That doesn't feel right.